src/parser/warrior/arms/modules/Abilities.js

```diff
@@ -177,14 +193,6 @@ class Abilities extends CoreAbilities {
         category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
         cooldown: 6,
         gcd: null,
-        castEfficiency: {
-          // TODO: Add a statistic to show how much damage was lost due to defensive stance
-          // TODO: Add a suggestion regarding having this up too much
-          // TODO: Add a suggestion regarding having this up too little, and then deactivate the cast efficiency since uptime is more accurate
-          suggestion: true,
-          recommendedEfficiency: 0.4,
-          extraSuggestion: 'If you\'re using this talent over something that passively increases your mobility or survivability, you better use it.',
-        },
         enabled: combatant.hasTalent(SPELLS.DEFENSIVE_STANCE_TALENT.id),
```

Comment:
Are the todos not relevant anymore if you've chosen the talent? If not, it's fine to remove them. If they are relevant, you might want to open an issue for them.

Reply:
The todos have been moved on the Defensive Stance file ('src\parser\warrior\arms\modules\talents\DefensiveStance.js), some have been already added in this file